async_service/nats_pub_sub.py

```diff
@@ -135,6 +150,22 @@ async def publish_output_message(
             timeout=5,
         )
 
+    async def get_output_message(
+        self, request_id: str, timeout: float = 0.5
+    ) -> Optional[bytes]:
+        jetstream = await self._get_js_client()
+        sub = await jetstream.subscribe(subject=f"{self._root_subject}.{request_id}")
```

Comment:
It is better if you explicitly set no-ack for this.

Reply:
https://github.com/nats-io/nats.py/blob/caaf42818eb80e180e40ae2225b49b9c47fd8b1d/nats/js/api.py#L333